.gitignore

```diff
@@ -1,7 +1,7 @@
 .DS_Store
 .atom/
 .dart_tool/
-.flutter-plugins-dependencies/
+.flutter-plugins-dependencies
```

Comment:
fwiw i definitely didn't type that in myself, so if this is wrong we probably have an error somewhere

Reply:
https://github.com/jocosocial/rainbowmonkey/pull/265 added it - it looks correct in the current template in the flutter repo, not sure what might have introduced it.